### PR TITLE
feat(psbt): add edge case tests

### PR DESCRIPTION
**Pull request by Nanosync**
#### What this PR does / why we need it:

Add tests
- There is an edge case where a newly added UTXO which is being used to cover the fee is equal to the fee that is being covered. However upon adding that UTXO, the size of the new input changes and requires more fees - this is not resolved yet and is skipped since we don't know the real cost of adding that UTXO.

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/sdk/src/transactions/__fixtures__/PSBTBuilder.fixture.ts
Comment (kevzzsk) [inline packages/sdk/src/transactions/__fixtures__/PSBTBuilder.fixture.ts:1]: I was thinking if we can create a map of addresses with responses. so on the mock function, we don't have to explicitly pass which response we want, but it will auto select based on the address passed to the mock function.

Comment (kevzzsk) [reply packages/sdk/src/transactions/__fixtures__/PSBTBuilder.fixture.ts:1]: This way, we can cater to more complex calls like if we have multiple outputs with different addresses, the mock can support them as well

Comment (Nanosync) [reply packages/sdk/src/transactions/__fixtures__/PSBTBuilder.fixture.ts:1]: > based on the address

do you mean the input address? i'm currently using same input address for all

Comment (kevzzsk) [reply packages/sdk/src/transactions/__fixtures__/PSBTBuilder.fixture.ts:1]: yes, so something like 
```js
export const unspentRes: Record<string, any> = {
  tb1p98dv6f5jp5qr4z2dtaljvwrhq34xrr8zuaqgv4ajf36vg2mmsruqt5m3lv: unspentsData1,
  tb1phduuvgaulft7xyt4zal284rxgjgah4nzf0huqk690fvcrc2229ss24ef72: unspentsData2,
};
```

Comment (kevzzsk) [reply packages/sdk/src/transactions/__fixtures__/PSBTBuilder.fixture.ts:1]: currently you are using one, but let's say we are to add more complex tests or this mock is also needed in other parts of the repo, then we should be able to reuse this mock.

one example is that, if we want to test creating PSBT with multiple inputs from different addresses (common for instantTrade)

File: packages/sdk/src/transactions/PSBTBuilder.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (kevzzsk) [inline packages/sdk/src/transactions/PSBTBuilder.ts:1]: i think you want to remove the no-console eslint and all the console logs as well

Comment (Nanosync) [reply packages/sdk/src/transactions/PSBTBuilder.ts:1]: oops this wasnt meant to be committed, removing
